themes/HeroesLounge-Theme/layouts/blog.htm

```diff
@@ -12,30 +12,22 @@
 <?php
 use RainLab\Translate\Models\Message;
 use Rikki\Heroeslounge\Models\Sloth;
+use Rikki\Heroeslounge\Classes\Helpers\TimezoneHelper;
 
 function onStart() {
     $this['baseUrl'] = Config::get('app.url');
     Message::setContext(App::getLocale(), $this->page->url);
+    $this['hasTimezone'] = TimezoneHelper::hasTimezone();
```

Comment:
I believe this is because of all the ajax.  Perhaps it is not setting the session key correctly in there somewhere.  Once the ajax has been removed, this can be revisited.  Right now, it is at least not worse than before, and should be better (less unnecessary calls).